.github/actions/force-image-digest/action.yml

```diff
@@ -0,0 +1,33 @@
+name: Force digest
+description: Force a digest for an image tag
+inputs:
+  tag:
+    description: Artifact to import
+  digest:
+    description: Digest to expect base image to be
+runs:
+  using: "composite"
+  steps:
+    - name: Force digest
+      uses: ./.github/actions/run-with-podman-service
+      with:
+        run: |
+          python <<'EOF'
+          import os
+          from make import DIGEST_CACHE_PATH
+          from make import _image_digests_write_cache
+          from _os import REPO
+          from _os import IMAGE
+          from _os.podman import podman
+          if not os.path.exists(DIGEST_CACHE_PATH):
+            print(f'No cache found at {DIGEST_CACHE_PATH}, creating new cache')
+
+          tag = os.environ["tag"]
+          digest = os.environ["digest"]
+          image = f"{REPO}:{tag}"
+          _image_digests_write_cache(image, digest)
+          podman("tag", digest, image)
```

Comment:
@coderabbitai any idea why the digest is not found? It is being passed in after being calculated in .github/workflows/build-variant.yaml in the id: image step

Reply:
It should be using the full `ghcr.io/eeems/atomic-arch:rootfs` tag no?
For some reason it's failing to find the image for the tag when trying to run ./make.py checkupdates in step id: build. Any ideas why that would be?

Reply:
Should I switch to skopeo copy since podman save/load doesn't preserve the digest?